x-pack/plugins/reporting/server/export_types/common/v2/get_full_redirect_app_url.test.ts

```diff
@@ -26,13 +26,20 @@ describe('getFullRedirectAppUrl', () => {
   });
 
   test('smoke test', () => {
-    expect(getFullRedirectAppUrl(reporting, 'test', undefined)).toBe(
-      'http://localhost:1234/test/s/test/app/reportingRedirect'
-    );
+    expect(
+      getFullRedirectAppUrl(reporting.getConfig(), reporting.getServerInfo(), 'test', undefined)
+    ).toBe('http://localhost:1234/test/s/test/app/reportingRedirect');
   });
 
   test('adding forceNow', () => {
-    expect(getFullRedirectAppUrl(reporting, 'test', 'TEST with a space')).toBe(
+    expect(
+      getFullRedirectAppUrl(
+        reporting.getConfig(),
+        reporting.getServerInfo(),
```

Comment:
Same as above: the server info and config should be mocked without a mockReportingCore